src/SFA.DAS.EmployerAccounts.Events/Messages/PaymentCreatedMessage.cs

```diff
@@ -1,25 +1,41 @@
 ﻿using SFA.DAS.Messaging.Attributes;
 using System;
+using SFA.DAS.Provider.Events.Api.Types;
 
 namespace SFA.DAS.EmployerAccounts.Events.Messages
 {
     [Serializable]
     [MessageGroup("payment_created")]
     public class PaymentCreatedMessage : AccountMessageBase
     {
-        public decimal Amount { get; protected set; }
-        public string ProviderName { get; protected set; }
+        public decimal Amount { get; set; }
```

Comment:
We've used protected setter before with no issues. Has this been changed because you have had any?